core-bundle/src/Resources/contao/library/Contao/StringUtil.php

```diff
@@ -971,6 +971,41 @@ public static function ampersand($strString, $blnEncode=true): string
 	{
 		return preg_replace('/&(amp;)?/i', ($blnEncode ? '&amp;' : '&'), $strString);
 	}
+
+	/**
+	 * Converts an input-encoded string back to its raw UTF-8 value it originated from.
+	 */
+	public static function getRawDecodedValue(string $strValue, bool $blnRemoveInsertTags = false): string
+	{
+		if ($blnRemoveInsertTags)
+		{
+			$strValue = static::stripInsertTags($strValue);
+		}
+		else
+		{
+			$strValue = Controller::replaceInsertTags($strValue, false);
```

Comment:
I don't like this because this should actually be a service so it cannot/should not reside in a static function :)